api/v1alpha1/servicebinding_types.go

```diff
@@ -96,7 +96,8 @@ type ServiceBindingSpec struct {
 	// BindAsFiles makes available the binding values as files in the application's container
 	// See MountPath attribute description for more details.
 	// +optional
-	BindAsFiles *bool `json:"bindAsFiles,omitempty"`
+	// +kubebuilder:default:=true
```

Comment:
@pedjak This is making the default value for `bindAsFiles` to `true`, this should be also reflected in the [UserGuide](https://github.com/redhat-developer/service-binding-operator/blob/master/docs/User_Guide.md#accessing-the-binding-data-from-the-application) where it is written that the default is `false`